src/main/java/nextmainfocus/util/Utility.java

```diff
@@ -0,0 +1,20 @@
+package nextmainfocus.util;
+
+import java.util.List;
+
+import org.json.JSONObject;
+import org.springframework.http.HttpStatusCode;
+
+import jakarta.annotation.Nullable;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
```

Comment:
For this method should be a private constructor

Reply:
`@UtilityClass` adds a private constructor <https://projectlombok.org/features/experimental/UtilityClass>